app/controllers/users_controller.rb

```diff
@@ -15,6 +16,11 @@ def update
     redirect_to user_path, notice: I18n.t(:user_data_updated)
   end
 
+  def accept_profile_terms
+    current_user.accept_profile_terms!
+    redirect_to root_path, notice: I18n.t(:terms_accepted)
```

Comment:
This redirection is not ideal but it'll be fixed by a subsequent PR

Reply:
It is not that bad